cdm/src/main/java/ucar/nc2/util/DebugFlagsImpl.java

```diff
@@ -71,4 +71,9 @@ public void set(String flagName, boolean value) {
     map.put(flagName, value);
   }
 
+  public String getSetFlags() {
+    String flags = map.keySet().toString();
+    return (flags.contains("${tds.debugflags}")) ? "" : flags;
```

Comment:
It's cruft from before I set the default property value in TdsContext. I'll update to remove it.